Python/pylifecycle.c

```diff
@@ -599,12 +599,9 @@ _Py_InitializeCore(const _PyCoreConfig *core_config)
 {
     assert(core_config != NULL);
 
-    PyInterpreterState *interp;
```

Comment:
This is just a question regarding the preferred C style for my learning -- from the code I've read, I noticed that most variable declarations are at the top of the function, but you've moved them to just before they are used (I assume for locality while reading). Is there a preference I should stick to?

Reply:
CPython is 28+ years old and has been written for ISO C90. Since Python 3.7, we are allowed to move variables declaration in the middle of functions: ISO C99 is allowed!!!

I don't recall why I decided to move these variables in this PR 😁 In general, we reject PR which are only coding style changes.